build/management/commands/build_arrestins.py

```diff
@@ -10,6 +10,7 @@
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
+from django.core.management.color import no_style
```

Comment:
If I'm not mistaken this commit was also part of the other PR right?

Reply:
yes. now merged ("rebased").